### ISSUE TITLE
Deprecation.

### ISSUE THREAD

**Issue by tmeasday** [tmeasday/create-graphql-server#55]
Note that I've not had time to spend on this project and I think the dependencies are increasingly falling out of date. 

If anyone is interested in taking over the project, please do! (@tobkle, @lorensr?)

Otherwise I've added a note: https://github.com/tmeasday/create-graphql-server/blob/master/README.md#create-graphql-server

**Comment by lorensr** [comment]
No, I found it very useful, though—thanks a lot for making it! Wonder in
future how much the "graphql database" idea of Prisma will take hold

**Comment by tobkle** [comment]
Thanks Tom for developing this. I still like the idea of having a NodeJS GraphQL generator and I also learnt a lot while working with it. Recently I was working with prisma, as there is a lot activity and they solved already a lot of topics, which this project would have to solve. What I don't like on prisma is, that it is in Scala, would have preferred a NodeJS solution. However, I cannot spend the required time on this project. Thanks for asking.
